features/270-next-skycli/commands.md

```diff
@@ -340,29 +339,42 @@ instructions again.
 
 After verification success. The domain will be usable immediately with letsencrypt ssl.
 
-### skycli domain update
+### skycli domain view
 
-`skycli domain update [CUSTOM_DOMAIN] --key=[KEY_FILE] --cert=[CERT_FILE] --use-letsencrypt`
+`skycli domain view [CUSTOM_DOMAIN]`
 
-Setup or update domain tls certificates. Cert and key should be PEM-encoded X.509, RSA (2048) key.
-Key and cert need to be provided at the same time.
+View the given domain, to retrieve the DNS records for verification.
 
-#### Flags
+#### Example
 
-- `--key=[KEY_FILE]` Provide custom tls key.
-- `--cert=[CERT_FILE]` Provide custom tls cert.
-- `--use-letsencrypt` Configure using let's encrypt certs for the given domain.
+```
+$ skycli domain view api.example.com
 
-### skycli domain set-alias
+General
 
-`skycli domain set-alias [CUSTOM_DOMAIN] [CF_VERSIONED_LINK]`
+  DOMAIN              VERIFIED         CUSTOM_CERT        SSL_CERT_EXPIRY               CREATED_AT
+  myapp.myapp.com     true             true               2020-11-26 20:00:00 +08:00    2019-11-26 18:00:00 +08:00
 
-Point the custom domain to specific CF version. Before setting the alias, the
-custom domain will point to the default domain `[APP_NAME].skygear.io`.
+DNS records
 
-#### Example
+  TYPE      HOST                     VALUE
+  TXT       _skygear.example.com     5636486ffc5a4dfebf4a13f480bd9a95
+  A         api.example.com          <ingress controller lb ip>
 
-`skycli domain set-alias api.example.com myapp.[HASH].skygear.io`
+```
+
+### skycli domain update
+
+`skycli domain update [CUSTOM_DOMAIN] --tls-secret=[SECRET_NAME] --use-letsencrypt`
+
+Update domain tls certificates and configure redirect, user can only provide either `tls-secret` or `use-letsencrypt`.
```

Comment:
I think you mean 'must provide either`.

Reply:
For the case developer want to update redirect only, they don't need to provide tls-secret or use-letsencrypt. Let me add back disable-redirect and redirect-domain to the command.